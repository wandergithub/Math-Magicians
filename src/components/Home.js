import Header from './Header';
import './styles/Home.css';

const Home = () => (
  <div>
    <Header />
    <div className="home-container">
      <h1 className="home-h">Welcome to our page!</h1>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit.
        Fusce eget pellentesque metus. Donec id tristique leo, vel convallis magna.
        Phasellus nulla leo,
        vestibulum nec sollicitudin eu, congue ac eros.
        Nam a magna felis. Aenean eget dui tincidunt,
        rhoncus nisi quis, consectetur orci. In maximus mi vel nisl dictum,
        quis consequat purus congue.
        Mauris dictum semper augue.
        Aenean mattis nisl in nulla tincidunt ultricies. Vivamus lacinia nisi nec lacinia ornare.
        Sed cursus nulla eget varius malesuada.
      </p>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit.
        Fusce eget pellentesque metus. Donec id tristique leo, vel convallis magna.
        Phasellus nulla leo,
        vestibulum nec sollicitudin eu, congue ac eros.
        Nam a magna felis. Aenean eget dui tincidunt,
        rhoncus nisi quis, consectetur orci. In maximus mi vel nisl dictum,
        quis consequat purus congue.
        Mauris dictum semper augue.
        Aenean mattis nisl in nulla tincidunt ultricies. Vivamus lacinia nisi nec lacinia ornare.
        Sed cursus nulla eget varius malesuada.
      </p>
    </div>
  </div>
);

export default Home;
