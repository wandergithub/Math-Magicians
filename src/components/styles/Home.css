.home-container {
  margin: 30px;
  margin-top: 50px;
}

.home-container > p {
  justify-self: center;
  width: 50%;
}
