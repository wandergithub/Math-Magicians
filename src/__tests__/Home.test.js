import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import Home from '../components/Home'

describe("Home page tests", () => {
  it("Renders correctly", () => {
    const tree = renderer
      .create(
        <MemoryRouter>
          <Home />
        </MemoryRouter>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
