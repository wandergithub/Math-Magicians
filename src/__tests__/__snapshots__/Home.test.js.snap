// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page tests Renders correctly 1`] = `
<div>
  <header
    className="nav-bar"
  >
    <h1>
      Math Magicians
    </h1>
    <div
      className="link-container"
    >
      <a
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
      <a
        className="mid-b"
        href="/Calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
      <a
        href="/Quote"
        onClick={[Function]}
      >
        Quote
      </a>
    </div>
  </header>
  <div
    className="home-container"
  >
    <h1
      className="home-h"
    >
      Welcome to our page!
    </h1>
    <p>
      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce eget pellentesque metus. Donec id tristique leo, vel convallis magna. Phasellus nulla leo, vestibulum nec sollicitudin eu, congue ac eros. Nam a magna felis. Aenean eget dui tincidunt, rhoncus nisi quis, consectetur orci. In maximus mi vel nisl dictum, quis consequat purus congue. Mauris dictum semper augue. Aenean mattis nisl in nulla tincidunt ultricies. Vivamus lacinia nisi nec lacinia ornare. Sed cursus nulla eget varius malesuada.
    </p>
    <p>
      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce eget pellentesque metus. Donec id tristique leo, vel convallis magna. Phasellus nulla leo, vestibulum nec sollicitudin eu, congue ac eros. Nam a magna felis. Aenean eget dui tincidunt, rhoncus nisi quis, consectetur orci. In maximus mi vel nisl dictum, quis consequat purus congue. Mauris dictum semper augue. Aenean mattis nisl in nulla tincidunt ultricies. Vivamus lacinia nisi nec lacinia ornare. Sed cursus nulla eget varius malesuada.
    </p>
  </div>
</div>
`;
